dnf/query.py

```diff
@@ -86,6 +86,14 @@ def na_dict(self):
     def pkgtup_dict(self):
         return per_pkgtup_dict(self.run())
 
+    def obsoleted_dict(self):
```

Comment:
Add the body of the function inside the examples file and document the `obsoletes`, etc. Package reldep attributes, please.